packages/schematics/schematics/schematic/files/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@angular-devkit/core": "^<%= coreVersion %>",
     "@angular-devkit/schematics": "^<%= schematicsVersion %>",
-    "typescript": "~3.8.2"
+    "typescript": "~3.9.2"
```

Comment:
Nit: might as well increase the patch version to the latest also.

Reply:
Done